SpywareFilter/sections/specific.txt

```diff
@@ -2853,7 +2853,7 @@ dreamprogs.net##td[width="88"][height="31"]
 !+ PLATFORM(ios, ext_safari, ext_android_cb)
 ||play.google.com/log?
 !+ NOT_PLATFORM(ios, ext_android_cb)
-||naver.net/wcslog.js$domain=~kyobobook.co.kr
+||naver.net/wcslog.js
```

Comment:
Please also explain this.

Reply:
I think what allowing the request that blocks EasyList by default in AdguardFilters is better.

Reply:
Ok, I got it.